lib/bolt/transport/winrm.rb

```diff
@@ -43,7 +44,8 @@ def self.default_options
           'connect-timeout' => 10,
           'ssl' => true,
           'ssl-verify' => true,
-          'file-protocol' => 'winrm'
+          'file-protocol' => 'winrm',
+          'basic-auth-only' => false
```

Comment:
This can be added to the `Bolt::Config::Transport::WinRM::DEFAULTS` hash: https://github.com/puppetlabs/bolt/blob/master/lib/bolt/config/transport/winrm.rb#L52-L57